src/bundles/index.js

```diff
@@ -20,13 +20,14 @@ import bundleCache from '../lib/bundle-cache'
 import ipfsDesktop from './ipfs-desktop'
 import repoStats from './repo-stats'
 import createAnalyticsBundle from './analytics'
+import experimentsBundle from './experiments'
 
 export default composeBundles(
   createCacheBundle(bundleCache.set),
   appIdle({ idleTimeout: 5000 }),
   ipfsBundle({
     tryWindow: false,
-    ipfsConnectionTest: async (ipfs) => {
+    ipfsConnectionTest: async ipfs => {
```

Comment:
We should let prettier do this kind of thing.

Reply:
Aren't we using [`eslint`](https://github.com/ipfs-shipyard/ipfs-webui/blob/master/package.json#L9)?

Reply:
My point is, I would like to agree the style guide and automate it rather than see these sorts of changes in PRs

Reply:
agree, I have prettier on by default but I don't think it respects the current `eslint` rules without additional config. I'll start a separate PR and we can sling our semicolons around there :)

Reply:
I'm resolving this conversation right now 😄 I've put the parenthesis back again. Thanks @cwaring! That'd be awesome